Marlin/src/HAL/STM32F1/MarlinSerial.h

```diff
@@ -49,28 +45,6 @@ struct MarlinSerial : public HardwareSerial {
       nvic_irq_set_priority(c_dev()->irq_num, UART_IRQ_PRIO);
     }
   #endif
-
```

Comment:
This shouldn't have been included in the bugfix branch. It's useless now, let's remove it.